internal/kubernetes/cert.go

```diff
@@ -47,8 +47,8 @@ func (v KubeletCertificateValidator) Run(ctx context.Context, informer validatio
 	defer func() {
 		informer.Done(ctx, name, err)
 	}()
-	if err = hybrid.ValidateCertificate(v.certPath, cluster.CertificateAuthority); err != nil {
-		err = hybrid.AddKubeletRemediation(v.certPath, err)
+	if err = certificate.ValidateCertificate(v.certPath, cluster.CertificateAuthority); err != nil {
```

Comment:
Since the thing you're validating is in the package name, I think we should change this to `Validate`
```suggestion
	if err = certificate.Validate(v.certPath, cluster.CertificateAuthority); err != nil {
```